### ISSUE TITLE
Add note about production deployment

### ISSUE THREAD

**Issue by ariddell** [aio-libs/aiohttp#234]
Users of the aiohttp server coming from bottle or flask may be concerned about performance and deployment (both bottle and flask have sections called "Deployment" or similar). If aiohttp's web server is fine to use in production, perhaps a note could be added to state this?

I'd be willing to perform a simple benchmark (against, say, cherrypy) but I suspect this has already been done. Has it?

For reference, this is the deployment section for bottle: http://bottlepy.org/docs/dev/deployment.html

**Comment by asvetlov** [comment]
You can use aiohttp in production.
We just run it under nginx (used as load balancer and static files handler, see http://nginx.com/resources/admin-guide/load-balancer/).

I have not finishing benchmarking but have semi-ready code in https://github.com/KeepSafe/aiohttp/blob/master/benchmark/async.py

**Comment by ariddell** [comment]
That's what I suspected. Maybe a rst "note" like this in the [high-level web server docs](http://aiohttp.readthedocs.org/en/v0.13.1/web.html): "Unlike the single-threaded HTTP servers common in Python before version 3.4, aiohttp's web server can be used in production deployments."

Or perhaps more detailed advice is needed? I know that Flask and Django have (very) popular modules or configurations where the WSGI app is configured to serve static files as well (e.g., Flask's serve_static and djstatic).

**Comment by ludovic-gasc** [comment]
Hi @ariddell,

For your information, we've 15+ aiohttp daemons on production, with a lot traffic, we have really better performances than before with Flask and Django.
We use HAproxy on some architectures, to handle TLS and distribute charge between several aiohttp daemons.

[ad] If you're interested in performance with aiohttp, I've made a small tool to build daemons with aiohttp, and distribute charge between several processes.
You can see a quick'n'dirty benchmark here: https://github.com/Eyepea/API-Hour#quickndirty-http-benchmarks-on-a-kitchen-table
You can follow the tutorial: http://pythonhosted.org/api_hour/tutorials/all_in_one.html
Don't hesitate to contact me if you have a question. [/ad]

Regards.

**Comment by asvetlov** [comment]
> Unlike the single-threaded HTTP servers common in Python before version 3.4

Why not? Twisted and Tornado are very widespread. aiohttp just gives simpler API for end users.

aiohttp.web _has_ support for static files, but it will be ready for production usage only after implementing `sendfile` in `asyncio`.

**Comment by ariddell** [comment]
This is great! I'm only suggesting that some of this information be in the docs because users of Flask, Django, and Bottle (unlike users of Twisted and Tornado) will anticipate specific instructions on Deployment, as the default for those Frameworks is single-threaded.

**Comment by asvetlov** [comment]
@ariddell as the issue initiator would you make a Pull Request with proposed documentation changes?

You know the best what you do like to see at result.

Ha, you are already done.

Thanks.

**Comment by fafhrd91** [comment]
i added simple section how to start aiohttp.web app with gunicorn https://github.com/KeepSafe/aiohttp/blob/master/docs/gunicorn.rst
